ci/update-homebrew-formula.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -eu -o pipefail
+
+[[ $# != 1 ]] && {
+  echo 1>&2 "USAGE: $0 <tag>"
+  exit 2
+}
+
+SRC_DIR="$(cd "${0%/*}/.." && pwd)"
+VERSION="${1:?}"
+TEMPLATE_FILE="${SRC_DIR}/pkg/brew/ripgrep-bin.rb.in"
+HOMEBREW_FILE="${TEMPLATE_FILE%.*}"
+
+OSX_FILE=ripgrep-${VERSION}-x86_64-apple-darwin.tar.gz
+LINUX_FILE=ripgrep-${VERSION}-x86_64-unknown-linux-musl.tar.gz
+URL_PREFIX=https://github.com/BurntSushi/ripgrep/releases/download/${VERSION}
+
+# shellcheck disable=2027,2064
+trap "rm -f $OSX_FILE $LINUX_FILE; exit 1" INT
+
+SLEEP_INTERVAL=5
+ROUND=0
+while ! [[ -f $OSX_FILE && -f $LINUX_FILE ]]; do
+  [[ $ROUND == 0 ]] && {
+    echo 1>&2 "Waiting for '$OSX_FILE' and '$LINUX_FILE' to become available... (Ctrl+C to interrupt)"
+  }
+  ROUND=$((ROUND + 1))
+  
+  for file in "$OSX_FILE" "$LINUX_FILE"; do 
+    [[ -f $file ]] && continue
+    { curl --fail -sLo "$file" "$URL_PREFIX/$file" \
+        && echo 1>&2 "Downloaded '$file'"; } || true
+  done
+  echo 1>&2 -n '.'
+  sleep $SLEEP_INTERVAL
+done
```

Comment:
What is the purpose of this loop? Why not just `curl` the two tar archives and call it a day?

Reply:
I think it's more easily understood when it's clear how I am using it.

My goal is to not have the feeling to have to wait for something.

I `git tag X` and push tags. Now travis will eventually produce the binaries, in any order. Instead of waiting around for the right time, I run `./ci/update-homebrew-formula.sh X` and just let it wait for me.

Eventually, after a long enough time, I go back and commit and push the updated brew file.
Something that might be useful here is to also `commit && push` in the script itself, so that it is truly fire & forget.